### ISSUE TITLE
Feature Request: Add ability to include/exclude specific ingress class names

### ISSUE THREAD

**Issue by halkeye** [ori-edge/k8s_gateway#254]
Some ingress controllers (like https://github.com/STRRL/cloudflare-tunnel-ingress-controller/issues/68) are not actually reporting ingress.Status.LoadBalancer.Ingress properly so k8s_gateway picks up the wrong ip.

I would like the abililty to exclude this ingress class so upstream dns can handle it.

**Comment by halkeye** [comment]
I was going to see if i couldn't make a quick PR, but I wasn't really sure of a great syntax for the config file.

Is this a good idea? Any ideas on format?

**Comment by networkop** [comment]
yeah, sounds like a good idea. UX could be similar to the existing `resources` stanza, i.e. explicit enumeration of watched ingress classes, defaulting to all if the list is empty.
